R/tm_missing_data.R

```diff
@@ -120,6 +122,7 @@ tm_missing_data <- function(label = "Missing data",
   )
 }
 
+# UI function for the missing data module
```

Comment:
does `ui_page_missing_data` have the same comment as `ui_missing_data`? similarly for the `server`

Reply:
One is for all datasets and latter is for single dataset